examples/md-flexible/tests/TimeDiscretizationTest.cpp

```diff
@@ -95,6 +95,27 @@ TEST_F(TimeDiscretizationTest, testCalculatePositions) {
   }
 }
 
+/**
+ * Test the mechanism that throws an exception when particles travel faster than skin/2/rebuildFreq
+ */
+TEST_F(TimeDiscretizationTest, testFastParticlesCheck) {
+  auto autoPas = std::make_shared<autopas::AutoPas<Molecule>>();
+  autoPas->setBoxMin({0., 0., 0.});
+  autoPas->setBoxMax({10., 10., 10.});
+  autoPas->setVerletSkinPerTimestep(.02);
+  autoPas->setVerletRebuildFrequency(10);
+  autoPas->init();
+
+  // slow particle -> no exception
+  autoPas->addParticle(autopas::MoleculeLJ({0., 0., 0.}, {0., 0., 0.}, 0));
```

Comment:
Would be worth making velocity non-zero (but still small).

Reply:
done 837a4e60